### ISSUE TITLE
Reported ImageNet results on test set or training set?

### ISSUE THREAD

**Issue by RuiFeiHe** [microsoft/robust-models-transfer#7]
Hi, impressed by your work! I'm curious that whether all the results reported in your paper's Table 3 is on ImageNet test set or training set?

**Comment by Hadisalman** [comment]
Hi @RuiFeiHe,

Thanks for your interest in our work! All the reported results are on the ImageNet validation set.

**Comment by RuiFeiHe** [comment]
Thanks! I'm closing this issue.
